docker/docker-compose-init-db.yml

```diff
@@ -0,0 +1,38 @@
+version: '3.2'
+services:
+
+  # ICUBAM service
+  db-builder:
+    build:
+      context: "${ICUBAM_COMPOSE_CONTEXT:-..}"
+      dockerfile: ./docker/Dockerfile
+    image: "${IMAGE_NAME:-icubam}:${IMAGE_TAG:-latest}"
+    container_name: db-builder
+    command: "./build_fake_dbs.sh"
+    tty: true
+    environment:
+      ENV_MODE : "$ENV_MODE"
+      SECRET_COOKIE : "$SECRET_COOKIE"
+      JWT_SECRET : "$JWT_SECRET"
+      GOOGLE_API_KEY : "$GOOGLE_API_KEY"
+      TW_KEY : "$TW_KEY"
+      TW_API : "$TW_API"
+      DB_SALT: "$DB_SALT"
+    working_dir: /home/icubam
+    volumes:
+      - type: bind
+        source: "${ICUBAM_CONFIG_PATH:-./resources/config.toml}"
+        target: /home/icubam/resources/config.toml
+      - type: bind
+        source: "${ICUBAM_COMPOSE_CONTEXT:-.}"
+        target: /home/icubam/db
+      - type: bind
+        source: "./build_fake_dbs.sh"
+        target: /home/icubam/build_fake_dbs.sh
```

Comment:
I had issues related to this related to the fact that the `project-directory` flag and the `context` are two different things that impact each other. With the provided setup and given the mount of the root of the project, I ended up with a folder named build_fake_dbs.sh at the root pf the project, and no db created.

When using docker-compose in `/home/pgraverdy/icubam` with project-directory value set to `.` and  build (`ICUBAM_COMPOSE_CONTEXT`) set to `..` then the build itself operates at  `/home/pgraverdy/icubam` (and not at `/home/pgraverdy/icubam/docker` if it was set to `.`) so the `./build_fake_dbs.sh` file is not found thus an empty folder is created by docker-compose.

Setting up ICUBAM_COMPOSE_CONTEXT to . allow the build part to process corretly, and since the script exists in the docker folder which is copied to the , we can just call it directly (no need to mount)


```suggestion
```